### ISSUE TITLE
Shovels/Swords unusable to build

### ISSUE THREAD

**Issue by moonbard** [Der1248/Castrum#1]
I have 3 stone shovels and 3 stone swords in my inventory but the build menu still says I don't have enough items. Tried putting swords/shovels in hotbar, as well as dropping on ground on build block. Seems like any build menu requiring tools won't take them.

Minetest Version 5.3.0, Castrum (game) version 1.11.2

**Comment by ApolloX939** [comment]
Yup... looks like they made their own tools/weapons. (They made their own so it's stackable)

Really should accept default tools/weapons as well as theirs. (Or make a recipe for making their tools, or even to convert default tools to theirs)

**Comment by ApolloX939** [comment]
A simple "fix" just do `/grantme give` then use `/giveme castrum:whatever_it_asks_for amount` then carry on.

Really this is just a workaround.

So for when it asks for stone shovels its `/giveme castrum:shovel_stone 3`,

like wise for the stone swords it's `/giveme castrum:sword_stone 3`.

**Comment by moonbard** [comment]
This turned out to be user error. **When clicking on the building with lava, there is a "craft" option.** I hadn't clicked on it again because I knew I didn't have the materials to upgrade it. Only realized other buttons were possible after I saw the "go to island" button on the pier.

Also, @Beanzilla , the items aren't in the game as castrum:shovel/sword - no name difference in creative mode.

**Comment by Der1248** [comment]
I will add more help about getting items. I don't what the normal craft in this game. Everything should be done with differend buildings. I will remove the normal craft in Castrum 1.12.0 and I will add something like a guide for the game.
